pyquil_for_azure_quantum/__init__.py

```diff
@@ -329,20 +305,34 @@ def run_batch(
                     f"{param_name} has length {len(param_values)} but {num_params} were expected."
                 )
 
+        executable = executable.copy()
         input_params = InputParams(
-            count=executable.num_shots, skip_quilc=executable.skip_quilc, substitutions=memory_map
+            count=executable.num_shots,
+            skip_quilc=executable.skip_quilc,
+            substitutions=memory_map,
         )
-
         job = self._target.submit(
             str(executable),
             name=name,
             input_params=input_params,
         )
-        combined_result = self.get_result(AzureJob(job, executable))
+        azure_job = AzureJob(job=job, executable=executable)
+        combined_result = self.get_result(azure_job)
         if num_params is None or num_params == 1:
             return [combined_result]
-        split_results = split(combined_result.readout_data["ro"], num_params)  # type: ignore
-        return [QAMExecutionResult(executable, {"ro": result}) for result in split_results]
-
 
-_RawData = Union[int, float, List[float]]
```

Comment:
Is this no longer used?

Reply:
Wasn't being used before or after the changes here 🤷